runtime/vm/ContinuationHelpers.cpp

```diff
@@ -152,6 +175,8 @@ enterContinuation(J9VMThread *currentThread, j9object_t continuationObject)
 	} else {
 		/* start new Continuation execution */
 		J9VMJDKINTERNALVMCONTINUATION_SET_STARTED(currentThread, continuationObject, JNI_TRUE);
+		continuation->state = VM_VMHelpers::setStartedToContinuationState(continuation->state);
```

Comment:
this API should operate with continuation:

VM_VMHelpers::setContinuationStarted(continuation);

(consistent with previous suggestion to introduce setContinuationFinished()